src/pandapipes/component_models/component_toolbox.py

```diff
@@ -159,7 +159,10 @@ def set_fixed_node_entries(net, node_pit, junctions, eg_types, p_values, t_value
                                     + press_sum) / (number + node_pit[index, eg_count_col])
         node_pit[index, type_col] = typ
         node_pit[index, eg_count_col] += number
-
+        if eg_type == 'p':
```

Comment:
somehow this part seems a bit lost here. I can't really find a perfect spot for it, but this does quite a bit of magic, which I am a bit sceptical of. It also means that some branch components will overwrite this value, if not well taken care of. Of course this happens with all entries touched in this function, but somehow I find hard to predict if this is completely safe to use with just some default value.

Reply:
I made it a bit more explicit and directly connected to each component!